Frontend/RecRock-Calendar-Frontend/src/app/calendar/components/toolbar/toolbar.component.ts

```diff
@@ -16,11 +16,14 @@ export class ToolbarComponent implements OnInit {
 
   public loginClass: string = this.SHOW_LOGIN;
```

Comment:
I would move the constants to a constants class. The naming convention is PascalCase for constants or even camelCase but not all caps with underscores